defaults/config.js

```diff
@@ -87,15 +87,15 @@ module.exports = {
 	prefetchStorage: false,
 
 	//
-	// Prefetch URLs Image Preview size limit
+	// Prefetch URLs media size limit
 	//
 	// If prefetch is enabled, The Lounge will only display content under the maximum size.
-	// Specified value is in kilobytes. Default value is 2048 kilobytes.
+	// Specified value is in kilobytes. Default value is 4096 kilobytes.
 	//
 	// @type     int
-	// @default  2048
+	// @default  4096
 	//
-	prefetchMaxImageSize: 2048,
+	prefetchMaxMediaSize: 4096,
```

Comment:
Instead of removing it, add `prefetchMaxMediaSize` on top of `prefetchMaxImageSize`. If `prefetchMaxImageSize` is present in the config, use it and warn that it is deprecated, otherwise use `prefetchMaxMediaSize`.
That way we don't break BC until v3, and we encourage people to switch :)

Reply:
I know it's slightly more complicated, but that's the cost of stability 😁

Reply:
Either that or the 2.7.0 PR contains just your first commit, but I think it's nice to give users some time to adapt peacefully lol, + it's just a matter of `config.prefetchMaxMediaSize = config.prefetchMaxImageSize;` within your `if` in `src/server.js`.

Reply:
well, it should still be removed from default config.js, right? Will make the swap elsewhere, though.

Reply:
Yes indeed, good point. So to keep bc you just need setting max media if there is a max image set

Reply:
Done

Reply:
I don't really see a reason to rename this since it's still going to be used only by images. Let's keep it this way until we actually need it for other purposes.

Reply:
It's used by the audio and video in #1806 and #1817

Reply:
It's actually not used by these because we came to the conclusion it's not need for 2 reasons:

1. Server aborts connection right after getting content-type.
2. Client only preloads metadata about the media so size doesn't matter in these cases.

Reply:
Oh, hmm, I missed that bit happening, but yeah, you are right.

Alright, I'm happy with either this going in or not going in. I guess it could be misleading if we have it as "media" when it's only images. But maybe users would think a gif isn't an image.